src/OrbitQt/CaptureOptionsDialog.cpp

```diff
@@ -30,56 +30,67 @@ CaptureOptionsDialog::CaptureOptionsDialog(QWidget* parent)
+                     ui_->memorySamplingPeriodMsLineEdit->setEnabled(checked);
+                     ui_->memoryWarningThresholdKbLabel->setEnabled(checked);
+                     ui_->memoryWarningThresholdKbLineEdit->setEnabled(checked);
                    });
-  QObject::connect(ui_->maxCopyRawStackSizeLineEdit, &QLineEdit::editingFinished,
-                   ui_->maxCopyRawStackSizeLineEdit, [this]() {
-                     if (ui_->maxCopyRawStackSizeLineEdit->text().isEmpty()) {
-                       ui_->maxCopyRawStackSizeLineEdit->setText(
-                           QString::number(kMaxCopyRawStackSizeDefaultValue));
-                     }
+
+  QObject::connect(ui_->samplingCheckBox, qOverload<bool>(&QCheckBox::toggled),
+                   ui_->samplingPeriodMsHorizontalLayout, [this](bool checked) {
+                     ui_->samplingPeriodMsLabel->setEnabled(checked);
+                     ui_->samplingPeriodMsDoubleSpinBox->setEnabled(checked);
+                     ui_->unwindingMethodGroupBox->setEnabled(checked);
                    });
 
-  ui_->unwindingMethodComboBox->addItem("DWARF", static_cast<int>(CaptureOptions::kDwarf));
-  ui_->unwindingMethodComboBox->addItem("Frame pointers",
-                                        static_cast<int>(CaptureOptions::kFramePointers));
-  ui_->maxCopyRawStackSizeLineEdit->setText(QString::number(kMaxCopyRawStackSizeDefaultValue));
-  ui_->dynamicInstrumentationMethodComboBox->addItem(
-      "Kernel (Uprobes)", static_cast<int>(CaptureOptions::kKernelUprobes));
-  ui_->dynamicInstrumentationMethodComboBox->addItem(
-      "Orbit", static_cast<int>(CaptureOptions::kUserSpaceInstrumentation));
-  if (!absl::GetFlag(FLAGS_devmode)) {
-    // TODO(b/198748597): Don't hide samplingCheckBox once disabling sampling completely is exposed.
-    ui_->samplingCheckBox->hide();
-    ui_->unwindingMethodWidget->hide();
-    ui_->maxCopyRawStackSizeWidget->hide();
-    ui_->schedulerCheckBox->hide();
-    ui_->gpuSubmissionsCheckBox->hide();
-    ui_->introspectionCheckBox->hide();
-  }
+  ui_->samplingPeriodMsLabel->setEnabled(ui_->samplingCheckBox->isChecked());
+  ui_->samplingPeriodMsDoubleSpinBox->setEnabled(ui_->samplingCheckBox->isChecked());
+  ui_->unwindingMethodGroupBox->setEnabled(ui_->samplingCheckBox->isChecked());
 
-  ui_->localMarkerDepthLineEdit->setValidator(&uint64_validator_);
-  ui_->memorySamplingPeriodMsLineEdit->setValidator(new UInt64Validator(
-      1, std::numeric_limits<uint64_t>::max(), ui_->memorySamplingPeriodMsLineEdit));
-  ui_->memoryWarningThresholdKbLineEdit->setValidator(&uint64_validator_);
-  ui_->maxCopyRawStackSizeLineEdit->setValidator(
-      new UInt64Validator(0, kMaxCopyRawStackSizeMaxValue, ui_->maxCopyRawStackSizeLineEdit));
+  ui_->framePointerUnwindingRadioButton->setToolTip(
+      "Frame-pointer unwinding has lower overhead, but requires all\n"
+      "binaries to be compiled with '-fno-omit-frame-pointer'.\n"
```

Comment:
> The other tooltips (again, in the .ui file) don't force a certain column width with extra newline characters.

Discussed offline: the other tooltips are much shorter, while this one would cause a very wide tooltip.